lib/webpacker/helper.rb

```diff
@@ -15,6 +16,28 @@ module Webpacker::Helper
   #   <%= javascript_pack_tag 'calendar', 'data-turbolinks-track': 'reload' %> # =>
   #   <script src="/packs/calendar-1016838bab065ae1e314.js" data-turbolinks-track="reload"></script>
   def javascript_pack_tag(name, **options)
-    javascript_include_tag(Webpacker::Source.new(name).path, **options)
+    javascript_include_tag(Webpacker::Source.new("#{File.basename(name, '.js')}.js").path, **options)
+  end
+
+  # Creates a link tag that references the named pack file, as compiled by Webpack per the entries list
+  # in config/webpack/shared.js. By default, this list is auto-generated to match everything in
+  # app/javascript/packs/*.js. In production mode, the digested reference is automatically looked up.
+  #
+  # Examples:
+  #
+  #   # In development mode:
+  #   <%= stylesheet_pack_tag 'calendar', 'data-turbolinks-track': 'reload' %> # =>
+  #   <link rel="stylesheet" media="screen" href="/packs/calendar.css" data-turbolinks-track="reload" />
+  #
+  #   # In production mode:
+  #   <%= stylesheet_pack_tag 'calendar', 'data-turbolinks-track': 'reload' %> # =>
+  #   <link rel="stylesheet" media="screen" href="/packs/calendar-1016838bab065ae1e122.css" data-turbolinks-track="reload" />
+  def stylesheet_pack_tag(name, **options)
+    webpacker_config = Webpacker::Configuration.webpacker
+    unless webpacker_config[:assets]
+      raise StandardError, "Stylesheet support isn't enabled. Install using - webpacker:install:assets"
+    end
+
+    stylesheet_link_tag(Webpacker::Source.new("#{File.basename(name, '.css')}.css").path, **options)
```

Comment:
The manifest generated basically looks like this, 

```json
{
  "counter.css": "/packs/counter-22c82e0c5245efbbb3bd.css",
  "counter.js": "/packs/counter-960a13cb514f03e64d74.js",
}
```
So, this is just to make sure we are passing correct name with extension for lookup, incase someone adds a `.css` extension to stylesheet pack tag, but perhaps that's over thought?